adwords/report.js

```diff
@@ -47,7 +47,8 @@ class AdwordsReport {
                 uri: 'https://adwords.google.com/api/adwords/reportdownload/' + apiVersion,
                 method: 'POST',
                 headers: headers,
-                form: this.buildReportBody(report)
+                form: this.buildReportBody(report),
+                timeout: 30 * 1000,
```

Comment:
Let's set the timeout in the AdwordsConstants so it can be changed before a request goes out.